docs/pages/examples.mdx

```diff
@@ -0,0 +1,10 @@
+import { Markdown } from '../components/Page';
+
+# Examples
+
+Keystone has a wide range of features, which are all highly configurable.
+To help show off these features in working projects our GitHub repository has a collection of [example projects](https://github.com/keystonejs/keystone/tree/master/examples).
+
+You can run all of these examples by cloning the Keystone repository and running the examples locally.
+
```

Comment:
```suggestion
Keystone has a wide range of highly configurable features.

Our [examples directory on GitHub](https://github.com/keystonejs/keystone/tree/master/examples) is a growing collection of projects you can run locally to learn more about a particular feature of Keystone.

Every project comes with its own docs that explain the how and why. They're useful as a reference for best practice, and as a jumping off point when adding features to your own Keystone project.
```

Reply:
Reframed this a little. Open to pushback!

Also / maybe ...  include a line about contributing your own example, and a link to `NEW-EXAMPLES.md` ?

Reply:
PS my diff suggestion isn't accurate. You'd have to remove a few lines above it. Not sure how to do that.

Reply:
> Also / maybe ... include a line about contributing your own example, and a link to NEW-EXAMPLES.md ?

We're not quite ready to accept contributions just yet.